recipe/bld.bat

```diff
@@ -0,0 +1,14 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_BUILD_TYPE=Release ^
+      ..
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```

Comment:
I thought just checking for the existence of the built libraries was sufficient?

I haven't put that piece in yet, because I don't know what the library names/locations will be. I just wanted to see it built on AppVeyor before worrying about that :smile: